lib/srv/desktop/rdp/rdpclient/Cargo.toml

```diff
@@ -20,7 +20,7 @@ num-traits = "0.2.15"
 rand = { version = "0.8.5", features = ["getrandom"] }
 rand_chacha = "0.3.1"
 rsa = "0.6.1"
-rdp-rs = { git = "https://github.com/gravitational/rdp-rs", rev = "6075679e7c9bd8e3c2136a87f097d05c7db3235f" }
+rdp-rs = { git = "https://github.com/gravitational/rdp-rs", rev = "004207e2edfbc6a57fd04daf6bcade93bcc3495e" }
```

Comment:
Please update this to latest master to ensure we get Lukasz's licensing work in v10.